skbio/metadata/tests/test_mixin.py

```diff
@@ -76,5 +79,40 @@ def __deepcopy__(self, memo):
         self._positional_metadata_constructor_ = ExamplePositionalMetadataMixin
 
 
+class TestIntervalMetadataMixin(unittest.TestCase, ReallyEqualMixin,
+                                IntervalMetadataMixinTests):
+    def setUp(self):
+        super()._set_up()
```

Comment:
This is a little confusing -- can you just define a `setUp` method in `IntervalMetadataMixinTests`, and call `super().setUp` here? That's usually how I've seen this handled.